src/roi/createRoi.m

```diff
@@ -146,6 +146,33 @@
 
       mask = createRoiLabel(mask);
 
+    case 'merge'
+
+      roiImages = specification;
```

Comment:
We are still developping this but I suspect we'll want to have the same way to use specification for all "cases":

so that it will be something like :

```
specification(1).mask = roi_filenameA
specification(2).mask = roi_filenameB
```

not necessary to have now but just to let you know that the "API" and the type and name of arguments for this function are still a moving target. :-)

Reply:
not sure if this replys to it but in a batch file you would have eg

```
roiImages = {fullfile(opt.ROI.roiDir, 'label-sphere5xMinus40yMinus50zMinus22_mask.nii'); ...
             fullfile(opt.ROI.roiDir, 'label-sphere5x36yMinus46zMinus17_mask.nii')};
```

Reply:
More like this

```
specification(1).mask = fullfile(opt.ROI.roiDir, 'label-sphere5xMinus40yMinus50zMinus22_mask.nii');
specification(2).mask = fullfile(opt.ROI.roiDir, 'label-sphere5x36yMinus46zMinus17_mask.nii');
```